### ISSUE TITLE
Pokemon Super Mystery Dungeon (eShop) Trouble

### ISSUE THREAD

**Issue by ashley-ketchum** [meladroit/svdt#21]
When I try to use svdt with my eShop copy of Pokemon Super Mystery Dungeon, I get a red screen. I'm on an old 3DS with 10-3.0.28U. In the Pokemon Picross issue, suloku said that games released after a certain date can't be dumped because of an issue with ninjhax that smealum has to fix. Is this one of those games?

**Comment by suloku** [comment]
Most likely, as it was released not long ago. I think the first game with this problem was triiforce heroes.

These games can't be booted under HANS either.

By the way cartridge versions should work just fine.

**Comment by dolopodog** [comment]
I have a cartridge version of Pokemon Super Mystery Dungeon and can't extract the save data either. The app will hang at a yellow screen during startup, and no automatic backup is generated.

Running a n3DS 9.2.0-20U. Using menuhax and browserhax both led to the same result.

**Comment by suloku** [comment]
I was told triiforce heroes cartridge version had no problems, maybe I was told wrong?

Probably now that 9.6+ keys are out hax 3.0 will fix this...

Maybe its because you are under 9.2? Have you tried using svdt under a 10.3 emunand?
You can launch hbl under emunand with this: https://github.com/yellows8/hblauncher_loader

**Comment by dolopodog** [comment]
Yes, the 9.6+ keys did fix the problem. The PSMD cart dumps fine running emunand 10.3.
